submissions/emekaAmadi/HTML-FORMS/contactForm.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact form page</title>
+  </head>
+  <body>
+    <main>
+      <form action="#" method="post" enctype="multipart/form-data">
+        <fieldset>
+          <label for="fullname">Full Name</label>
+          <input
+            type="text"
+            id="fullname"
+            name="fullname"
+            placeholder="Enter full name"
+          />
+          <br /><br />
+          <label for="email">Email</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="Enter your email address"
+          />
+          <br />
+          <br />
+          <label for="subject">Subject</label>
+          <input
+            type="text"
+            id="subject"
+            name="subject"
+            placeholder="Enter Subject"
+          />
+          <br /><br />
+          <label for="message">Message</label>
```

Comment:
Message should have a type textarea with rows and  columns to accommodate message

Reply:
ok ma